### PR TITLE
Add width parameter

### LINKED ISSUE #65: Expose a width parameter for daq.Tank (and maybe height instead of size)
At the moment the width of a `daq.Tank` is hardcoded (see https://github.com/plotly/dash-daq/blob/master/src/styled/Tank.styled.js#L8). We could also expose `height` instead of the current parameter `size` which is a bit confusing. Raised by https://community.plot.ly/t/edit-dash-daq-component-size/28394

### PR DESCRIPTION
**Pull request by shammamah**
Closes #65. 

(Please note that 4612d58cf62adbebf4cc190443ce457b48946727 and all subsequent commits have to do with https://github.com/plotly/dash-daq/issues/34)

### CODE REVIEW

File: src/components/Tank.react.js [hunk 1]
```diff
@@ -71,7 +72,8 @@ const Tank = props => {
 Tank.defaultProps = {
   min: 0,
   max: 10,
-  size: 192,
+  height: 192,
+  width: 150,
```
Comment (alexcjohnson) [inline src/components/Tank.react.js]: is it intentional to change this from 112 that was the default before?

Comment (shammamah) [reply src/components/Tank.react.js]: Good catch! Fixed in c357158.

File: src/components/Tank.react.js [hunk 2]
```diff
@@ -56,13 +57,13 @@ const Tank = props => {
...
-            {showCurrentValue && currentValue}
-          </TankContainer>
-        </Container>
+	<Container>
+	  {scaleContainer}
+	  <TankContainer height={height} width={width}>
+	    <TankFill color={color} height={`${percentageFill}%`} />
+	    {showCurrentValue && currentValue}
+	  </TankContainer>
+	</Container>
```
Comment (alexcjohnson) [inline src/components/Tank.react.js]: why the indentation change?

Comment (alexcjohnson) [reply src/components/Tank.react.js]: oh it looks like these are hard tabs? Let's try not to use those, just spaces.

File: src/components/Thermometer.react.js [hunk 3]
```diff
@@ -68,20 +67,20 @@ const Thermometer = props => {
...
+	      <TankFill
+		thermometer
+		color={color}
+		height={`${computeProgress({ min, max, value })}%`}
+	      />
+	      {theme.dark ? null : <Bulb on={value !== min} color={color} />}
+	    </TankContainer>
+	  </Container>
+	  {showCurrentValue && currentValue}
+	</ThermometerContainer>
```
Comment (alexcjohnson) [inline src/components/Thermometer.react.js]: same here - no hard tabs

**Pull request merged**